floodwatch/src/js/components/FilterParent.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import $ from 'jquery';
+import {GraphParent} from './GraphParent';
+import _ from 'lodash';
+import d3 from 'd3'
+
+type Props = {
+  side: string;
+  currentTopic: string;
+  updateMouseOver: Function;
+}
+
+export class FilterParent extends Component {
+  props: Props;
+
+  constructor(props: Props) {
+    super(props);
+  }
+
+  stackData(data) {
+    let topics = Object.keys(data);
```

Comment:
Suggest using const for 100% of variable that don't change - here and elsewhere.